### PR TITLE
Helpers: Supress warning for PDF module and increment the version

### PR DESCRIPTION
**Pull request by LukasRanorex**

### CODE REVIEW

File: AutomationHelpers/test/packages.config [hunk 1]
```diff
@@ -3,5 +3,5 @@
   <package id="NSubstitute" version="2.0.3" targetFramework="net45" />
   <package id="NUnit" version="3.8.1" targetFramework="net45" />
   <package id="NUnit.ConsoleRunner" version="3.7.0" targetFramework="net45" />
-  <package id="Ranorex.ReportToPDF" version="1.0.16" targetFramework="net45" />
+  <package id="Ranorex.ReportToPDF" version="1.0.17" targetFramework="net45" />
```
Comment (ahoisl) [inline AutomationHelpers/test/packages.config]: According to nuget, there is no 1.0.17 version out, yet:
https://www.nuget.org/packages/Ranorex.ReportToPDF/

File: AutomationHelpers/src/Modules/ReportToPDFModule.cs [hunk 2]
```diff
@@ -282,6 +282,7 @@ internal void Cleanup()
 			}
 		}
 
+#pragma warning disable CS0618
```
Comment (ahoisl) [inline AutomationHelpers/src/Modules/ReportToPDFModule.cs]: Why not put the pragma directly around the UseScreenshotFolder?

**Comment by ahoisl** [comment]
@lstoiser Why did the warnings not show up in the auto-tests? I guess they don't check for warnings, but IMHO they should

**Comment by lstoiser** [comment]
> @lstoiser Why did the warnings not show up in the auto-tests? I guess they don't check for warnings, but IMHO they should

**Comment by lstoiser** [comment]
@ahoisl right, we only check for build errors, not for warnings. But I'll extend automation to also check for warnings

**Comment by ahoisl** [comment]
BTW, there's a typo in the commit message, it should be "Suppress" :-)

File: AutomationHelpers/src/Modules/ReportToPDFModule.cs (hunk 2 above)
Comment (LukasRanorex) [reply AutomationHelpers/src/Modules/ReportToPDFModule.cs]: Isn't that ugly to put the Pragma to the method directly?

Comment (ahoisl) [reply AutomationHelpers/src/Modules/ReportToPDFModule.cs]: I would limit it to the minimum context so we see other warnings

**Pull request merged**